src/common/MoviePeople/index.js

```diff
@@ -9,7 +9,8 @@ export const MoviePeople = ({ title, items }) => {
         {items &&
           items.map((item) => (
             <PersonTile
```

Comment:
To mozna zapisać tak
`items?.map((item)) => {}`
Jest krócej i bardziej czytelniej, a działa tak samo.

Reply:
w porządku skróciłem zapis